### DISCUSSION TITLE
Query raw body even size

### DISCUSSION THREAD

**Discussion by Tthongsy** [datalust/seq-tickets#1264] [Help]
Is there a way to search events based on char size of an event or by the body size?

I tried reading through the documentation and reading the Seq-cheat-sheet but I couldn't find anything like that.

**Comment by nblumhardt** [comment]
Hi @Tthongsy  - yes, you can do this with `length(tojson(@Data))`, e.g.:

```
length(tojson(@Data)) > 1000
```

Keep in mind that it's an expensive operation (requires re-serialization of the event fields into JSON), so should only be used sparingly :-)

Let me know if this helps!
Nick

**Reply by Tthongsy** [reply]
Hi, thanks for the help.
I'm very new to this, I wasn't sure if this would work because I'm trying to also understand as I was reading the ingestion log. There was a "Rejecting payload from "x.x.x.x" due to oversized event; first 1024 chars of 40234". Would I be able to find this through Seq or is this just not going to be found in Seq and only found in the ingestion log?

Thanks,
Tommy

**Reply by nblumhardt** [reply]
Hi Tommy, thanks for your reply.

Events only end up in the ingestion log when they can't be ingested into Seq's event stream, so you won't find these events by searching there.

The first 1024 chars should give you a clue as to the kind of event; unfortunately there's not a lot more detail saved than that, though.

Let me know if there's anything we can do to help track it down.

Best regards,
Nick

**Reply by Tthongsy** [reply]
Hi Nick,
I tried using length(tojson(Data)) > 1000.
I keep getting an error that tojson was not recognized.

**Reply by Tthongsy** [reply]
Actually, nevermind. I got it solved.

Thanks,
Tommy
